signature/algorithm.go

```diff
@@ -11,12 +13,12 @@ type Algorithm int
 // One of following supported specs
 // https://github.com/notaryproject/notaryproject/blob/main/signature-specification.md#algorithm-selection
 const (
-	AlgorithmPS256 Algorithm = 1 + iota // RSASSA_PSS_SHA256
-	AlgorithmPS384                      // RSASSA_PSS_SHA384
-	AlgorithmPS512                      // RSASSA_PSS_SHA512
-	AlgorithmES256                      // ECDSA_SHA256
-	AlgorithmES384                      // ECDSA_SHA384
-	AlgorithmES512                      // ECDSA_SHA512
+	AlgorithmPS256 Algorithm = 1 + iota // RSASSA-PSS with SHA-256
+	AlgorithmPS384                      // RSASSA-PSS with SHA-384
+	AlgorithmPS512                      // RSASSA-PSS with SHA-512
+	AlgorithmES256                      // ECDSA on secp256r1 with SHA-256
+	AlgorithmES384                      // ECDSA on secp384r1 with SHA-384
+	AlgorithmES512                      // ECDSA on secp521r1 with SHA-512
```

Comment:
Should we rename `AlgorithmES512` to `AlgorithmES521`?  Because the actual signing alg name is `secp521r1 `.